tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "target": "es5",
     "typeRoots": ["./types", "./node_modules/@types"]
   },
-  "include": ["src/**/*"],
+  "include": ["src"],
+  "exclude": ["dist", "node_modules"],
```

Comment:
Is `execlude` necessary? it it for workaround on bundler?

Reply:
This is necessary otherwise I'll see this:

```
Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: infinite/dist/src/utils/resolve-args.d.ts.
The file must be included in at least one of the projects provided
```